style.css

```diff
@@ -40,3 +40,127 @@ a:hover {
                 ****** GLOBAL ******
     ==============================================
 */
+
+.top-nav {
+  background-color: #f72210;
+}
+
+.time {
+  color: #fff;
+}
+
+.login {
+  color: #fff;
+}
+
+.subscribe {
+  background-color: #ff0;
+  border-radius: 25px;
+  color: #000;
+}
+
+.subscribe:hover {
+  color: #000;
+}
+
+.login:hover {
+  color: #fff;
+}
+
+.nav-head button {
+  height: 20px;
+  font-size: 15px;
+  line-height: 30px;
+  padding: 0 10px;
+}
+
+.hidden-user {
+  color: #fff;
+}
+
+.nav-head img {
+  width: 120px;
+}
+
+.middle-nav,
+.bottom-nav {
+  border-bottom: rgb(190, 190, 190) solid 1px;
+}
+
+.middle-nav a {
+  color: black;
+  border-right: rgb(190, 190, 190) solid 1px;
+}
+
+.last-link {
+  border: none !important;
```

Comment:
Remove important

Reply:
Thanks. We have removed this and used an alternative with id tag styling.